src/main/resources/static/css/layout.css

```diff
@@ -237,5 +238,17 @@ html, body, .spinner {
 }
 
 
+.payment-list-item {
```

Comment:
duplicate class!!! set the template div to use .search-filters instead. If only one line differs in the future, create a seperate class to override the value. If at the end of the ride it'll differ too much, then a seperate class is entirely justified

Reply:
removed, will be in DEV-013